tests/nlu/featurizers/test_regex_featurizer.py

```diff
@@ -350,3 +350,50 @@ def test_regex_featurizer_case_sensitive(
     sequence_featrures, sentence_features = ftr._features_for_patterns(message, TEXT)
     assert np.allclose(sequence_featrures.toarray()[0], sequence_vector, atol=1e-10)
     assert np.allclose(sentence_features.toarray()[-1], sentence_vector, atol=1e-10)
+
+
+@pytest.mark.parametrize(
+    "sentence, expected, labeled_tokens, use_word_boundaries",
```

Comment:
@samsucik do you think it's too aggressive to change all of them? I felt it was confusing at first too and then kind of forgot as I got used to it

Reply:
No, I don't think it's too aggressive. If we genuinely believe that one pattern is more clear than the other (and otherwise they're equivalent), then I would go for it. My only concern here is that in general once someone starts making changes aimed to improve code clarity, it can take too much of their time 🙂

Reply:
>take too much of their time

Your time now too as the code review got larger :S. I changed all the tests with this `expected` pattern. That way people won't re-use it for new tests in the future.